bin/varnishtest/tests/m00017.vtc

```diff
@@ -335,3 +337,40 @@ client c6 {
 } -run
 
 server s1 -wait
+
+# this could work, but would need additional coding to save
+# the right snapshot of the bereq on the req ws
+varnish v1 -errvcl {Not available in subroutine 'vcl_pipe'} {
```

Comment:
I was surprised by this one and surveyed the `bereq` symbols not available in `vcl_pipe`. I think that `bereq.xid` could be made available since there technically is a transaction tied to `bereq`. I would rather see this land in this patch series since this is squaring the pipe semantics.

Reply:
done